docs/resources/faq.md

```diff
@@ -599,6 +599,39 @@ Chia versions 2.1.0 and newer no longer support the version 1 (v1) blockchain da
 6. Verify the config file (~\.chia\mainnet\config\config.yaml) has the correct value under the full_node section for `database_path: db/blockchain_v2_CHALLENGE.sqlite` (should only need to change the v1 to v2).
 7. Launch chia and wait for a bit (the height to hash, sub-epoch, and peers files need to be built so this can take 5-10 minutes).
 
+### How can I connect to the same full node peers whenever I start Chia?
+
+:::info
+
+This functionality was first made available in Chia version 2.2.0, in [PR #17369](https://github.com/Chia-Network/chia-blockchain/pull/17369).
```

Comment:
Should we add a note that Felix contributed this functionality?

Reply:
No reason not to